task/task-1.py

```diff
@@ -0,0 +1,11 @@
+from utils.reader import read_file
+
+list_data = read_file()
+for data in list_data:
+    date = data.split(",")[0]
+    max_temp = data.split(",")[1]
+    min_temp = data.split(",")[3]
+    diffirance = int(max_temp) - int(min_temp)
+
+
```

Comment:
never have two new lines spaces. only one line space is sufficient

Reply:
ok